console/src/components/Common/utils/v1QueryUtils.ts

```diff
@@ -98,6 +98,42 @@ export const getDropPermissionQuery = (
   };
 };
 
+export const getInsertQuery = (
+  tableDef: TableDefinition,
+  insertObject: object
+) => {
+  const queryCols = Object.keys(insertObject);
+  const colString = queryCols.join(',');
+
+  const queryValues = Object.values(insertObject);
+  const modifiedValues = queryValues.map(value => {
+    // currently, only modifying for string data,
+    // unsure of how it is for other types.
+    if (typeof value === 'string') {
+      return `'${value}'`;
+    }
+
+    return value;
+  });
+  const valueString = modifiedValues.join(',');
+
+  // NOTE:
```

Comment:
A point of discussion

Reply:
Also, a reminder to throw more light on the `type: "insert"` through the errors that I got.

Reply:
`executeSQL` is also available